packages/shared/src/util/features.ts

```diff
@@ -468,3 +468,207 @@ export function validateAndFixCondition(
   }
   throw new Error("Invalid targeting condition JSON: " + res.error);
 }
+
+export function getDefaultPrerequisiteCondition(
+  parentFeature?: FeatureInterface
+) {
+  const valueType = parentFeature?.valueType || "boolean";
+  if (valueType === "boolean") {
+    return `{"value": true}`;
+  }
+  return `{"value": {"$exists": true}}`;
+}
+
+export function isPrerequisiteConditionConditional(
+  condition: string,
+  prerequisiteState: PrerequisiteState
+) {
+  if (
+    [`{"value": {"$exists": false}}`, `{"value": {"$exists": true}}`].includes(
+      condition
+    )
```

Comment:
We should ideally be matching on the parsed condition, not the stringified version.  Otherwise, it's super fragile when it comes to whitespace.

Reply:
replaced with evalCondition()